src/utp_stream.cpp

```diff
@@ -89,6 +88,29 @@ void utp_log(char const* fmt, ...)
 	va_end(l);
 }
 
+bool is_utp_stream_logging() {
+	return log_file_holder.utp_log_file != NULL;
+}
+
+void set_utp_stream_logging(bool enable) {
+	if (enable)
+	{
+		if (log_file_holder.utp_log_file == NULL)
+		{
+			log_file_holder.utp_log_file = fopen("utp.log", "w+");
+		}
+	}
+	else
+	{
+		if (log_file_holder.utp_log_file != NULL)
+		{
+			FILE* f = log_file_holder.utp_log_file;
+			log_file_holder.utp_log_file = NULL;
+			fclose(f);
+		}
+	}
+}
+
 #define UTP_LOG utp_log
 #if TORRENT_VERBOSE_UTP_LOG
```

Comment:
sorry to nit pick (and I should have pointed this out in my first comment). In a way I think it's better to change the source since libtorrent isn't necessarily built from the Jamfile. One could argue that the difference is just some warnings. On the other hand though, some distros have policies of warning free builds (one of the BSDs at least) and one of my longer term goals is to be able to build with -Werror (or warnings=error).

does that make sense?